WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -774,7 +790,6 @@ nick_the_philosopher = {
 			trait = imbecile
 			trait = inbred
 			trait = incapable
-			religion = catholic
 			religion_group = pagan_group
```

Comment:
No it doesn't. It pushes the historical revisionist idea that Catholicism was a bastion of ignorance that resisted all progress until the Rennaisance. It never did anything like that unless its power was directly threatened or what it saw as the integrity of the faith was at stake. It's simply and woefully inaccurate. There were plenty of Catholic philosophers over the centuries.